.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup working Directory
+        run: |
+          sudo mkdir -p /usr/src/extension
+          sudo cp -R ./* /usr/src/extension
+          cd /usr/src/extension
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '17'
+
+      - run: |
+          mkdir -p artifacts
+          echo "" > artifacts/file.txt
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: Artifacts
+          path: artifacts/file.txt
```

Comment:
There's another VS Code extension repo that uses GHA to build and upload the .visx file and attach it to a Github release. That build is here https://github.com/Workiva/intl-extension-vscode/blob/master/.github/workflows/build.yaml

Can you use that as a reference to  update this workflow? Looks like there are some slight differences in this repo, like using yarn instead of npm. You can add a "package" command to package.json just like here https://github.com/Workiva/intl-extension-vscode/blob/master/package.json#L80

Reply:
Updated the workflow as per the suggestions. Following are some changes:

1. For upload artifacts action - can't use private actions as no self-hosted runners available.
2. Can't use yum to install xvfb as ubuntu-latest runner supports apt-get not yum. So updated the same.